oe_content.install

```diff
@@ -37,3 +39,17 @@ function oe_content_update_8001() {
     $entity_definition_update_manager->updateFieldStorageDefinition($field_definition);
   }
 }
+
+/**
+ * Update field storage with oe_reference.
+ */
+function oe_content_update_8002() {
+  // Obtain configuration from yaml files.
+  $storage = new FileStorage(drupal_get_path('module', 'oe_content') . '/config/install');
+
+  // Obtain the storage manager for field storage bases,
+  // Create a new field from the yaml configuration and save.
+  \Drupal::entityManager()->getStorage('field_storage_config')
+    ->create($storage->read('field.storage.node.oe_reference'))
+    ->save();
```

Comment:
Let's use Drupal API to create the base field:

```php
use Drupal\field\Entity\FieldStorageConfig;

<?php
    FieldStorageConfig::create([
      'field_name' => 'oe_reference',
      'entity_type' => 'node',
      ....
    ])->save();
```

Reply:
Fixed